index.html

```diff
@@ -61,21 +65,17 @@
 							box-shadow: 0px 3px 2px rgba(0, 0, 0, 0.2);
 						}
 
-						paper-tabs::shadow #selectionBar {
+						paper-tabs #selectionBar {
```

Comment:
Hallo,

bitte schmeißen Sie meine Email-Adresse von Ihre List raus, ich bekomme jeden Tag unendlich solche Emails von dieser Adresse, und nur in Spam wegschmeißen bringt es gar nicht.

vielen Dank

Date: Sun, 8 Nov 2015 13:14:17 -0800
From: notifications@github.com
To: todomvc@noreply.github.com
Subject: Re: [todomvc] Homepage Polymer 1.1 update ✨ (#1507)

In index.html:

> @@ -61,21 +65,17 @@
>  							box-shadow: 0px 3px 2px rgba(0, 0, 0, 0.2);
>  						}
>  
> -						paper-tabs::shadow #selectionBar {
> +						paper-tabs #selectionBar {

this will break under native shadow dom. Please use the css mixin (--paper-tabs-selection-bar-color)


https://elements.polymer-project.org/elements/paper-tabs


—
Reply to this email directly or view it on GitHub.